src/views/Event/index.js

```diff
@@ -84,8 +135,12 @@ class EventPage extends React.Component {
             )
         }
 
-        const editEventButton = <Button raised onClick={e => this.editEvent(e)} disabled={!eventIsEditable} color="primary"><FormattedMessage id="edit-event"/></Button>
+        const eventDeletabilityExplanation = 'You do not have rights to delete this event.';
```

Comment:
I would just use `eventEditabilityExplanation` here in cancel and delete buttons too, because 1) they are already localized in Finnish like the rest of the UI, and 2) because they explain why the event cannot be edited in more detail. Basically the same rights apply for delete and cancel as for edit, so it is alright to tell the user they cannot edit the event even in delete and cancel.